### ISSUE TITLE
Validation mIoU metric monitor

### ISSUE THREAD

**Issue by meetshah1995** [meetshah1995/pytorch-semseg#2]
Port tensorflow code (from private repo) / write code to show mIoU over validation dataset every few epochs.

**Comment by meetshah1995** [comment]
solved by 2f797b4df1320234adcfde3b069f646747b5113e
